### ISSUE TITLE
Circle strokeWidth does not behave like SVG

### ISSUE THREAD

**Issue by chris-ck** [software-mansion/react-native-svg#2178]
# Question

I am trying to animate a Circle inward, since it doesn't work with box-shadow as in CSS, the idea is to render a circle and animate r from progress.value * radius down to 1 and stroke-width from 0 up to radius. The desired animation is:

https://github.com/software-mansion/react-native-svg/assets/9639670/ec9fa435-c97f-4a55-b2bf-c0d92ab73666

and is done in CSS as simple as:

```css
@keyframes inward {
  0% {
    r: 50;
    stroke-width: 0;
  }
  
  100% {
    r: 1;
    stroke-width: 98;
  }
}
```

This could potentially work in react-native-svg too, however, react-native-svg fails to render the stroke properly as soon as the radius - strokeWIdth / 2 becomes lower than zero.

HTML/CSS: 

```HTML
    <circle
        cx="50%"
        cy="50%"
        fill="transparent"
        stroke="#ff0000"
        r="1"
        stroke-width="50"
      />
```

renders

<img width="72" alt="Captura de ecrã 2023-11-27, às 02 28 06" src="https://github.com/software-mansion/react-native-svg/assets/9639670/99851d90-ccf7-415d-92b9-eab5b2d54711">

The following in react-native-svg:

```JSX
           <Circle
              cx="50%"
              cy="50%"
              fill="transparent"
              stroke="#ff0000"
              r="1"
              strokeWidth="50"
            />
```

<img width="77" alt="Captura de ecrã 2023-11-27, às 02 27 48" src="https://github.com/software-mansion/react-native-svg/assets/9639670/fc1c98bd-a0c3-4eef-a50a-e1fc721bf410">

Long story short: can I achieve the desired effect in react-native-svg?

Any help much appreciated.

Cheers

**Comment by chris-ck** [comment]
Just to be clear: I do have a workaround for my current issue at hand:
```JS
interpolate(progress.value, [0, 1], [radius, radius * 0.5])
```

leads to my desired effect, however, this might become an issue in other places where a workaround like this is not possible. Is this as expected or should I open a bug ticket for this?

Cheers

**Comment by bohdanprog** [comment]
Hello @chris-ck,

Unfortunately, react-native-svg does not currently support CSS keyframes. We have noted this as a potential feature to add to our roadmap. If you have any further questions, please feel free to open a new issue.

Thank you.
